### ISSUE TITLE
OpenJDK 10 fails with 404 error

### ISSUE THREAD

**Issue by peschkaj** [AdoptOpenJDK/homebrew-openjdk#7]
```
brew install adoptopenjdk-openjdk10
==> Installing adoptopenjdk-openjdk10 from adoptopenjdk/openjdk
==> Downloading https://github.com/AdoptOpenJDK/openjdk10-releases/releases/download/jdk-10%2B23/OpenJDK10_x64_Mac_

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "adoptopenjdk-openjdk10"
Download failed: https://github.com/AdoptOpenJDK/openjdk10-releases/releases/download/jdk-10%2B23/OpenJDK10_x64_Mac_jdk-10.23.tar.gz
```

**Comment by karianna** [comment]
Couple of issues here:

1. That Release no longer exists (was purged due to a security concern
2. The brew code should probably download from our API as opposed to GitHub directly.

Use can manually change the URL inside the brew config to:

https://github.com/AdoptOpenJDK/openjdk10-releases/releases/download/201807101745/OpenJDK10_x64_Mac_201807101745.tar.gz

**Comment by karianna** [comment]
Fixed by #8
